post_office/admin.py

```diff
@@ -201,6 +206,19 @@ def fetch_email_image(self, request, pk, content_id):
                 return HttpResponse(message.get_payload(decode=True), content_type=message.get_content_type())
         return HttpResponseNotFound()
 
+    def resend(self, request, pk):
```

Comment:
This is definitely a feature that's nice to have. However I'd like to change the behavior a little bit.

Instead of creating a new email, it should just fetch the current email and do an `email.dispatch()`, essentially just doing a resend on the email.

Reply:
absolutely!